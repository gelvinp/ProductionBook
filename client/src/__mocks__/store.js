const store = jest.fn()
export default store
