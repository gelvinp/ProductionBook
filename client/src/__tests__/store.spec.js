import store from '../store.js'

describe('createStore', () => {
  it('should initialize correctly', () => {
    expect(store).toMatchSnapshot()
    expect(store.getState()).toMatchSnapshot()
  })
})
