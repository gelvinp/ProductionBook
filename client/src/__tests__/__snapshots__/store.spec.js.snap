// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createStore should initialize correctly 1`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(observable): [Function],
}
`;

exports[`createStore should initialize correctly 2`] = `
Object {
  "documents": Object {
    "selectedDocument": Object {
      "section": -1,
      "uuid": -1,
    },
  },
  "modals": Object {
    "document": Object {},
    "section": -1,
    "uploadOpen": false,
  },
  "password": "",
  "sections": Object {},
}
`;
